main/cluster/predict.py

```diff
@@ -26,19 +26,15 @@ def configure(configname=None):
             except Exception as exc:
                 # print (tb.format_exc())
                 print (exc)
-    # print(configuration)
     conf=Struct(**configuration)
     conf.TRACEBACK=True
     conf.VERBOSE=True
     if not config.configured:
         config.configure(conf) # configure() receives a python module
     assert config.configured
-    # for statement in config.IMPORTS:  #FIXME is this ugly code?
```

Comment:
Remove this. It was there because I thought lacking imports were the origin of issue #8, but it's not the case.